domains/arabiancoconut.is-cool.dev.json

```diff
@@ -3,14 +3,14 @@
     "description": "The domain for project use",
 
     "domain": "is-cool.dev",
-    "subdomain": "arabiancoconut",
+    "subdomain": ["arabiancoconut","bio.arabiancoconut"],
 
     "owner": {
-        "email": "p11h0ifd@duck.com"
+        "email": "zwar73e7@duck.com"
     },
 
     "record": {
-        "CNAME": "arabiancoconut.github.io"
+        "CNAME": ["arabiancoconut.github.io","bioflask.onrender.com"]
```

Comment:
CNAME cannot be an array. Must be single string value.

Reply:
Got it I will fix it, and can I only have 1 CNAME what if I want multiple CNAMES?